cmd/really/really.cpp

```diff
@@ -137,9 +138,16 @@ class ReallyServer : public quicr::ServerDelegate
 
     qtransport::TransportConfig tcfg{ .tls_cert_filename = "./server-cert.pem",
                                       .tls_key_filename = "./server-key.pem" };
-    server =
-      std::make_unique<quicr::QuicRServer>(relayInfo, tcfg, *this, logger);
+    server = std::make_unique<quicr::Server>(
+      relayInfo, tcfg, shared_from_this(), logger);
+  }
+
+public:
+  static std::shared_ptr<ReallyServer> create()
+  {
+    return std::shared_ptr<ReallyServer>(new ReallyServer());
```

Comment:
```suggestion
    return std::make_shared<ReallyServer>();
```

Reply:
When this code was still in there that wouldn't have compiled since the constructor was private (see https://en.cppreference.com/w/cpp/memory/enable_shared_from_this). Regardless, this has been removed.